packages/codeeditor/src/editor.ts

```diff
@@ -806,7 +806,7 @@ export namespace CodeEditor {
     fontFamily: null,
     fontSize: null,
     lineHeight: null,
-    lineNumbers: false,
+    lineNumbers: true,
```

Comment:
Please do not change the default setting.

```suggestion
    lineNumbers: false,
```

Reply:
I had done this as a quick fix to this issue where as we see the `lineNumbers: true` is set in System Defaults but it by default the line numbers is actually false -
![image](https://user-images.githubusercontent.com/41439841/131813013-1242cc2c-86b7-4b4b-9033-46c2c921b5cf.png)

Reply:
You are right - this was an inconsistency. And now the UI test are failing because your fix is removing the inconsistency by changing the default.
Sorry for the wrong comment, could you revert to `lineNumbers: true` so the default behavior stays the same (aka the editor displays line numbers by default).

Reply:
I see, alright!

Reply:
The js-codeeditor and js-codemirror error are legit. Could you update the test to reflect the change in the default setting value.

Reply:
Yes sure. Done!

Reply:
Ok this is really tricky because they are the default for the cells and the text editor. But what I don't get is that they should be overridden by the settings in order to hide the line number in cells by default and to show them for the text editor.

Could you look into that?

For notebook, settings are loaded (and set) at https://github.com/jupyterlab/jupyterlab/blob/c13cebd0afbe99cce715371489f7c7eb2ec78a11/packages/notebook-extension/src/index.ts#L1103
For text editor, it is at https://github.com/jupyterlab/jupyterlab/blob/c13cebd0afbe99cce715371489f7c7eb2ec78a11/packages/fileeditor-extension/src/index.ts#L120

Reply:
Hey @fcollonval, I have a little trouble understanding what needs to be done here. Do we need to look why the UI says `lineNumbers` is `true` while it is `false` in defaultConfig for the text editor?

Reply:
Oh maybe you are talking about the predefined configuration for `User Preferences`? I actually have overwritten the User Preferences. How can I reset the user preferences to the absolute configuration provided by default with a fresh installation?

Reply:
Hey @Mithil467 sorry for the late reply. To clean your user settings, you need to remove the settings in %USERPROFILE%\.jupyter\lab\user-settings (if on Windows), $HOME/.jupyter/lab/user-settings (otherwise).

So yes we need to figure out why the default configuration for all editors (in the notebook and in the text file) are taken instead of the default settings that should allow to have line numbers in text file by default but not in notebook cells.

Reply:
I see! I already deleted the mentioned `~/.jupyter` folder and then when I see the user preferences it is empty.
![image](https://user-images.githubusercontent.com/41439841/132134233-c3b8ad95-94a5-48c1-b7f5-9c3343233fb7.png)

Reply:
Apparently there is no file called `lab\user-settings` inside the `.jupyter` folder

Reply:
The user preferences are not set by default on a fresh installation as per the gitter conversation with @jasongrout. I guess for moving ahead we can either
1. Set this to `false` -
https://github.com/jupyterlab/jupyterlab/blob/cd577621f1d8a5b6bbfb5d1912d38fa3855cb96a/packages/fileeditor-extension/schema/plugin.json#L164
2. Or update the galata tests to reflect this change.
I guess the second option will help resolving this https://discourse.jupyter.org/t/permanently-activate-line-numbers-in-editor/3864/3 too.